### DISCUSSION TITLE
Nest.js+Next.js+Typesciprt project can not be deployed to the Vercel

### DISCUSSION THREAD

**Discussion by topdevhere623** [vercel/community#339] [Show & Tell]
Hello, everyone!
I just deployed my app built on Nest(Node.js Framework)+Next.js to the Vercel.
But after completion of the deployment the app seems to can not be connected to the backend, so the login and registration is shows the error.
I really need in your help!
Please ping me if you're great man in this fields.

**Comment by Pieparker** [comment]
Hey there! Your post is a little vague.

Does the error occur locally? Can you elaborate on the error further? Can you provide URLs to the deployment, or the project ID? Does the project fail to deploy or is the error observed as part of a web request? What _is_ the backend? What _is_ the error?

Thanks!

**Reply by topdevhere623** [reply]
Thank your reply!
https://next-nest-linkforme.vercel.app/logup
I already deployed to the Vercel but it is not working with the backend, so I can not neither log in nor register.

**Reply by topdevhere623** [reply]
But it was working on local perfectly with all functionalities!
The curious thing was that It was not working with the command "yarn start:next", it should be "yarn start:dev" to be fully activated with the backend in my local.
But how can the Vercel launch the app with "yarn start:dev"?
![image](https://user-images.githubusercontent.com/93789798/153427788-a6a33648-5c45-4969-85d3-c554038d2334.png)

**Reply by Pieparker** [reply]
Sounds like you should reconsider your `scripts` to better suit what's happening! [Here](https://vercel.com/docs/concepts/deployments/build-step#install-command) is how you can alter your install command.

**Reply by topdevhere623** [reply]
Thank you!
Can you suggest this in details for me?
I don't get it watching your link.

**Reply by Pieparker** [reply]
If you're unfamiliar with your own code base, I'd suggest starting from scratch and understanding the principles, before jumping in with both feet 😉. To update the `Install command`, this is achieved in your project's UI.

**Reply by topdevhere623** [reply]
Thank you!
I'll try it!
